src/ssl.c

```diff
@@ -17352,6 +17352,7 @@ long wolfSSL_set_tlsext_status_ocsp_resp(WOLFSSL *s, unsigned char *resp,
     if (s == NULL)
         return WOLFSSL_FAILURE;
 
+    XFREE(s->ocspResp, NULL, 0);
```

Comment:
`0`? What is the correct dynamic type?

Reply:
This is tricky. The ocspResp is allocated by the application. I'm following the openSSL implementation here that frees the old value when setting a new one but we can't set any dynamic type as we don't allocate the ocspResp in the first place.